src/main/resources/dataset-registration-schema_v1.json

```diff
@@ -355,6 +356,12 @@
           "label": "Publication Moratorium (MOR)",
           "description": "Publication Moratorium (MOR)"
         },
+        "morDate": {
+          "type": "string",
+          "format": "date",
+          "label": "Publication Moratorium Date (MOR)",
+          "description": "Publication Moratorium Date (MOR)"
+        },
```

Comment:
right now in the code merged into the UI, the date is saved in the `mor` field, and is undefined if the user does not check it. easy fix, but would need to be changed

Reply:
Yes, I think that was a sort of workaround. When I reviewed the mocks, we needed two fields, a boolean for the field and then a date value for an actual date. If the user unselected/toggled the boolean option between saves, we could remember the original date selection.

<img width="369" alt="Screen Shot 2022-09-29 at 4 05 24 PM" src="https://user-images.githubusercontent.com/116679/193131548-b921a34d-23b7-4c19-8a70-36b323949b94.png">

Reply:
Also, there was a slack conversation that clarified this to some degree.

Reply:
ahhh yeah, I think I achieved remembering it by keeping state outside of the formData, in a react state variable. I could see arguments for either, though